device/manager.go

```diff
@@ -321,6 +321,22 @@ func (m *manager) readPump(d *device, r ReadCloser, closeOnce *sync.Once) {
 			m.measures.RequestResponse.Add(1.0)
 		}
 
+		// update message metadata fields
+		if message.Metadata == nil {
+			message.Metadata = map[string]string{}
+		}
+		message.Metadata["partner-ids"] = strings.Join(event.Device.PartnerIDs(), ",")
+		message.Metadata["session-id"] = event.Device.SessionID()
```

Comment:
Do we really need this?  Remember that metadata is supposed to be for a device's RDK stack to populate.  We use this for connect/disconnect events in talaria as hack, since those events don't come from devices.